app/assets/javascripts/components/menu_item.js.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var MenuItem = React.createClass({
+
+  navigate: function(hash) {
+    window.location.hash = hash;
+  },
+
+  render: function() {
+    var classStyle = "menu-item";
+    if (this.props.hash === "share-facebook" || this.props.hash === "get-started") {
```

Comment:
After I change all of this and make a commit, do I need to submit a new PR?

On Tue, May 5, 2015 at 5:14 PM, Braden Anderson <notifications@github.com>
wrote:

> In app/assets/javascripts/components/menu_item.js.jsx
> <https://github.com/scalableinternetservices/U1F44D/pull/11#discussion_r29726506>
> :
>
> > @@ -0,0 +1,19 @@
> > +var React = require('react');
> > +
> > +var MenuItem = React.createClass({
> > +
> > +  navigate: function(hash) {
> > +    window.location.hash = hash;
> > +  },
> > +
> > +  render: function() {
> > +    var classStyle = "menu-item";
> > +    if (this.props.hash === "share-facebook" || this.props.hash === "get-started") {
>
> No need to comment, just make your changes in the next commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scalableinternetservices/U1F44D/pull/11/files#r29726506>
> .
>